cpp/vm/evmzero/interpreter.cc

```diff
@@ -1487,6 +1489,22 @@ struct CreateImpl {
     if (gas -= mem_cost; gas < 0) [[unlikely]]
       return {.dynamic_gas_costs = initial_gas - gas};
 
+    if (ctx.revision >= EVMC_SHANGHAI) {
+      const int64_t max_code_size = 24576;
+      const int64_t max_init_code_size = 2 * max_code_size;
+      const int64_t init_code_word_cost = 2;
```

Comment:
When starting the `evmzero` the agreement was to follow Google's C++ style guide. Not sure to what extend that was done, but one part of it would ask constants to be prefixed with a `k` followed by mixed-case (see [guidlines](https://google.github.io/styleguide/cppguide.html#Constant_Names)). So those constants should be named `kMaxCodeSize`, `kMaxInitCodeSize`, and `kInitCodeWordCost`.

Reply:
Why doesn't clang-format deal with this ?
IMO, we should not need to manually deal with these details. 
I offer to write a patch about that.

Reply:
@LuisPH3 do you think clang format is powerful enough to identify such cases?

Reply:
I was thinking something in this direction:
https://clang.llvm.org/extra/clang-tidy/checks/readability/identifier-naming.html
(sorry, this is part of clang-tidy, not format)